act/accordions.md

```diff
@@ -0,0 +1,93 @@
+# Accordions
+
+## Rule Identifier
+
+gel-foundations-accordions
+
+## Rule Description
+
+The accordion consists of a number of collapsed sections, each with a button to expand that section's content.
+
+This rule tests that when an accordion is included in an HTML page it meets the requirement of the BBC GEL Accordion component.
+
+## Rule Type
+
+atomic
+
+## Accessibility Requirements Mapping
+
+- [BBC GEL Accordions](https://bbc.github.io/gel/components/accordions/)
+  - Required for conformance to BBC GEL
+  - Outcome mapping:
+    - Any failed outcomes: success criterion is not satisfied
+    - All passed outcomes: success criterion needs further testing
+    - An inapplicable outcome: success criterion needs further testing
+
+## Input Aspects
+
+DOM Tree
+CSS Styling
+
+## Applicability
+
+This rule applies to any HTML page that includes a BBC GEL Accordion component.
+
+## Expectations
+
+@NOTE: This assumes we are documenting a valid Accordion, not specifically the [Accordion reference implementation](https://bbc.github.io/gel/components/accordions/#reference-implementation)
```

Comment:
The practice of explicitly describing the scope of the document, as you have done here, is very helpful in my opinion. Shall we do this for all documents?

Reply:
I'd left this in as a note to you because I think the reference implementation documentation misses a few points, I've added requirements here that aren't specified in https://bbc.github.io/gel/components/accordions/#reference-implementation. This might be something for the _Background_ section mentioned in the Headings pull request.